### ISSUE TITLE
Blocking Call to load_default_certs in SmartThinq Sensors Causes Performance Issues in Home Assistant

### ISSUE THREAD

**Issue by cloudbr34k84** [ollo69/ha-smartthinq-sensors#793]
Describe the Bug
The custom integration smartthinq_sensors is making a blocking call to ssl.create_default_context() inside the Home Assistant event loop, which leads to performance degradation. The operation to load default SSL certificates should be asynchronous to prevent blocking the event loop.

Expected Behavior
The integration should load SSL certificates asynchronously without causing blocking operations that affect Home Assistant's performance.

Screenshots
No applicable screenshots for this issue.

Environment Details:

Environment (HASSIO, Raspbian, etc): HASSIO
Home Assistant version installed: 2024.9.1
Component version installed: 0.39.2
Last known working version: Not sure
LG device type and model with issue: RC90V9_AUS-Dryer (DRYER), F_V7_Y___W.A__QEUK-FL (WASHER)
by LG

Logs
```
Logger: homeassistant.util.loop
Source: util/loop.py:136
First occurred: 3:40:10 pm (1 occurrences)

Detected blocking call to load_default_certs with args (<ssl.SSLContext object at 0x7f0ef7dff0d0>, <Purpose.SERVER_AUTH: _ASN1Object(nid=129, shortname='serverAuth', longname='TLS Web Server Authentication', oid='1.3.6.1.5.5.7.3.1')>) inside the event loop by custom integration 'smartthinq_sensors' at custom_components/smartthinq_sensors/wideq/core_async.py, line 149: context = ssl.create_default_context() (offender: /usr/local/lib/python3.12/ssl.py, line 713: context.load_default_certs(purpose)), please create a bug report at https://github.com/ollo69/ha-smartthinq-sensors/issues
```

**Comment by McGiverGim** [comment]
I can confirm the issue, I have seen the same error message.

**Comment by McGiverGim** [comment]
I think this PR fixes the issue? https://github.com/ollo69/ha-smartthinq-sensors/pull/791

If yes, this this is a duplicate of https://github.com/ollo69/ha-smartthinq-sensors/issues/790

**Comment by cloudbr34k84** [comment]
Ok will wait, thankyou

**Comment by ollo69** [comment]
Closed by #791
